app/views/admin/invoices/show.html.erb

```diff
@@ -1,8 +1,19 @@
 <h1>Admin Invoice - <%= @invoice.id %></h1>
-<p>Status: <%= @invoice.status %></p>
-<p>Created at: <%= @invoice.created_at.strftime('%A, %B %d, %Y') %></p>
-<p>Customer: <%= @invoice.customer.first_name %> <%= @invoice.customer.last_name %></p>
 
+  <%= form_with(model: @invoice, url: admin_invoice_path(@invoice), method: :patch) do |form| %>
+    <div>
+      <label for="invoice_status">Invoice Status:</label>
+      <%= form.select :status, options_for_select(Invoice.statuses.keys.map { |status| [status.titleize, status]}, @invoice.status) %>
+      <%= form.submit 'Update Invoice Status' %>
+    </div>
+  <% end %>
+
+  <p>Created at: <%= @invoice.created_at.strftime('%A, %B %d, %Y') %></p>
```

Comment:
This might be too much logic for the view

Reply:
I did not think of that. Per your suggestion I moved it over to the model.

Reply:
I might push that change of date logic back up to the PR. I'm not seeing it in the model or the view.

Reply:
ooooooh, wait, I made an invoice method to that stringifies the date. Call `@invoice.formatted_date`

Reply:
Brilliant that worked like a charm and I updated this